test/sets.jl

```diff
@@ -147,6 +147,10 @@ s = union(Set([5,6,7,8]), Set([7,8,9]))
 s = Set([1,3,5,7])
 union!(s,(2,3,4,5))
 @test isequal(s,Set([1,2,3,4,5,7]))
+@test ===(typeof(union(Set([1]), IntSet())), Set{Int})
```

Comment:
Will do next time. I just copied this style from the beginning of `test/sets.jl`. I think I should I do such stylistic changes to otherwise unrelated lines of code? That would complicate review and merging...